app/components/post_draft/send_action/index.tsx

```diff
@@ -34,6 +41,14 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
             alignItems: 'center',
             justifyContent: 'center',
         },
+        scheduledPostTooltipStyle: {
+            shadowColor: '#000',
+            shadowOffset: {width: 0, height: 2},
+            shadowRadius: 2,
+            shadowOpacity: 0.16,
+            width: 250,
+            height: 140,
```

Comment:
aren't you missing `elevation` for android ?

Reply:
Done.